pyannote/audio/pipelines/clustering.py

```diff
@@ -200,6 +232,18 @@ def __init__(self, metric: str = "cosine", expects_num_clusters: bool = False):
             ["Affinity", "Unnormalized", "RandomWalk", "GraphCut"]
         )
         self.eigengap = Categorical(["Ratio", "NormalizedDiff"])
+        self.spectral_min_embeddings = Categorical([5, 10])
+
+        # Hyperparameters for refinement operations.
+        self.refinement_sequence = Categorical(
+            ["", "TS", "GTS", "CGTSDN"])
+        self.gaussian_blur_sigma = Uniform(0, 3)
+        self.p_percentile = Uniform(0, 1)
```

Comment:
Those two hyper-parameters are never used in `__call__`.
I am pretty sure this is **not** what you had in mind :-)

Reply:
Just pushed a fix for that.

Reply:
Nice catch! I must have accidentally removed them while debugging...